### DISCUSSION TITLE
Would this be workable on WSA?

### DISCUSSION THREAD

**Discussion by Crav3ndale** [dorssel/usbipd-win#221] [Q&A] [answered]
Just saw this great project and wondered if it would be workable on Windows Subsystem for Android? They both run a Linux kernel, and so would I be able to follow the same steps to connect a USB device to android or would the code need changing? Thanks!

**Comment by dorssel** [comment]
I don't know Windows subsystem for Android, but if it has a USPIP client it will probably work. It also works with remote (bare metal) Linux and Windows machines, so why not? This project is not just for WSL, it is a generic USBIP compatible server.

**Comment by dorssel** [comment, marked as answer]
See:
https://github.com/WSA-Community/WSA-Linux-Kernel/blob/92470dc4ae48d89196a540df16f305b18f46265a/configs/wsa/config-wsa-5.10#L3413

Standard WSA kernel: No.
And I don't know if you can replace it with your own compiled kernel like you can with WSL (I think not).
